content/memos/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Memos
+html_description: Some examples of what Open Terms Archive exposes, as identified by independent contributors.
+hero:
+  title: Memos
+  subtitle: Some examples of what Open Terms Archive exposes, as identified by independent contributors.
```

Comment:
Can you use non breakable spaces in Open Terms Archive to avoid line break in the middle of the name

Reply:
Same here, remove `.`